src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -823,4 +828,22 @@ private function getBulkCategoriesFromRequest(Request $request)
 
         return $categoryIds;
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return bool
+     */
+    private function requestHasSearchParameters(Request $request)
+    {
+        if (!$request->query->has('category')) {
+            return false;
+        }
+
+        if (!array_key_exists('filters', $request->query->get('category'))) {
```

Comment:
Unnecessary conditions, the `!empty` is already doing theses checks

```suggestion
    private function requestHasSearchParameters(Request $request)
    {
        return !empty($request->query->get('category')['filters']);
    }
```

is enough